web/styles/compose.css

```diff
@@ -319,6 +319,17 @@
         gap: 0 6px;
     }
 
+    .messagebox:has(.new_message_textarea:focus) {
+        #message-formatting-controls-container {
+            border-color: color-mix(
+                in srgb,
+                var(--color-message-content-container-border-focus) 75%,
```

Comment:
Can you explain and document in comments where the percentages (75% here, 60% below) are coming from? I worry I may have steered you wrong with color-mix, as we have an opaque color on `--color-message-content-container-border-focus` for light mode. The semi-opaque dark-mode color probably needs a treatment like this, but I don't understand the origin of these percentages.